osu.Game.Rulesets.Mania/Skinning/Legacy/ManiaLegacySkinTransformer.cs

```diff
@@ -75,6 +77,22 @@ public override Drawable GetDrawableComponent(ISkinComponent component)
         {
             switch (component)
             {
+                case SkinnableTargetComponent targetComponent:
+                    if (targetComponent.Target == SkinnableTarget.MainHUDComponents)
+                    {
+                        var components = base.GetDrawableComponent(component) as SkinnableTargetComponentsContainer;
+                        if (components == null) return null;
+
+                        foreach (var comboSplash in components.OfType<LegacyComboSplash>())
+                        {
+                            comboSplash.BurstCondition = combo => combo > 0 && combo % 100 == 0;
+                        }
+
+                        return components;
+                    }
+
+                    break;
```

Comment:
Rather than doing this, it will be cleaner if you make a `ManiaLegacyComboSplashSide` and return it below, and then include the conditional logic within the subclass.